monitors/SyntheticPegMonitor.js

```diff
@@ -34,28 +39,36 @@ class SyntheticPegMonitor {
 
     // Default config settings. SyntheticPegMonitor deployer can override these settings by passing in new
     // values via the `config` input object. The `isValid` property is a function that should be called
-    // before resetting any config settings. `isValid` must return a Boolean.
+    // before resetting any config settings. `isValid` must return a Boolean. If the associated price feed is missing
+    // then the defaults to 0 thresholds. This will skip the check in the respective functions.
     const defaultConfig = {
       deviationAlertThreshold: {
         // `deviationAlertThreshold`: Error threshold used to compare observed and expected token prices.
-        // if the deviation in token price exceeds this value an alert is fired. If set to zero then fire no logs.
-        value: 0.2,
+        // If the deviation in token price exceeds this value an alert is fired. If set to zero then fire no logs.
+        value: uniswapPriceFeed && medianizerPriceFeed ? 0.2 : 0,
         isValid: x => {
-          return x < 100 && x >= 0;
+          return x != null && x < 1 && x >= 0;
```

Comment:
nit: couldn't you rewrite this as `return x && x < 1 && x >= 0`?

Reply:
I originally had it like this but there is a pretty good reason why I changed to check for the `null` case: if you set this check to `x && x < 1 && x >= 0` then this will return false if `x=0` meaning that you cant set it to 0 as this will be deemed invalid, even though it's valid for the input & inequality.

I actually think that a better check might be to simply say `return typeof x === 'number' && x < 1 && x >= 0`. Thoughts?

Reply:
I have implemented this check for a specific type. I think this is best as it is consistent with the other validations.

Reply:
Ahh great call

Reply:
+1 one on the `typeof`